### ISSUE TITLE
Missing backslash

### ISSUE THREAD

**Issue by paul356** [nikomatsakis/rust-etags#4]
https://github.com/nikomatsakis/rust-etags/blob/75e721079b26da9c2c6527b2780dbc87fb39ee63/ctags.rust#L7

You may need  a backslash for { in the pattern.

I meet such a warning
```
ctags: Warning: regcomp macro_rules! +([a-zA-Z0-9_]+) *{: Unmatched \{
```

**Comment by nikomatsakis** [comment]
Ha! I haven't used this in years, but you may well be right. I assume somebody has a more recent version by now...

**Comment by paul356** [comment]
Yes. There is another repo dan-t/rusty-tags for generating tags for rust. If you don't mind I can create a PR for you.
